README.md

```diff
@@ -155,6 +155,8 @@ functions](https://golang.org/pkg/text/template/#hdr-Functions)):
 | `color`         | `color.Color, string` | Wrap the text in color (.ContainerColor and .PodColor provided)                   |
 | `parseJSON`     | `string`              | Parse string as JSON                                                              |
 | `tryParseJSON`  | `string`              | Attempt to parse string as JSON, return nil on failure                             |
+| `extractJSONParts`  | `...string`       | Parse string as JSON and concatenate the given keys.                                     |
+| `tryExtractJSONParts` | `string`        | Attempt to parse string as JSON and concatenate the given keys. , return text on failure |
```

Comment:
I think these arguments should be `string, ...string` as the first argument is JSON text.

Reply:
Done